packages/@okta/vuepress-site/docs/guides/migrate-to-okta-bulk/main/index.md

```diff
@@ -1,22 +1,31 @@
 ---
-title: Bulk Migration with Credentials
+title: Bulk migration with credentials
+meta:
+  - name: description
+    content: Perform a bulk migration of users into Okta using the Okta API
+layout: Guides
 ---
 
-## Performing a migration using the Okta API
-
 As part of your plan and preparation to migrate your users to Okta, you gathered your source data into an intermediate staging area such as a secure local database or a CSV file. This guide shows you how to use that data to create users and groups in Okta using the [Okta Users API](/docs/reference/api/users/) and the [Okta Groups API](/docs/reference/api/groups/).
 
-### Prerequisites
+---
+
+**Learning outcomes**
+
+Perform a bulk migration of users into Okta using the Okta API.
```

Comment:
There is more than one API they can use, but they don't use all of the Okta APIs.
```suggestion
Perform a bulk migration of users into Okta using Okta APIs.
```